index.html

```diff
@@ -1,3 +1,29 @@
-<html>
-  <h1>hello main</h1>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="styles.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <header class="header">
+        <h1>
+            My Logo
+        </h1>
+        <a href="#">Portfolio</a>
+        <a href="#">About</a>
+        <a href="#">Contact</a>
+    </header>
+    <section class="mainsection">
+        <div class="text-section">
+            <p class="text1">I’m Bunbee <br>Glad to see you!</p>
```

Comment:
- [ ] The main title (e.g., "I'm Bunbee ...") is a h1 tag, refactor with `h1` not `p` tag